src/TortoiseGit.ps1

```diff
@@ -11,60 +11,74 @@ function private:Get-TortoiseGitPath {
 
 $Global:TortoiseGitSettings = new-object PSObject -Property @{
   TortoiseGitPath = (Get-TortoiseGitPath)
+  TortoiseGitCommands = @{
+    "about" = "about";
+    "add" = "add";
+    "blame" = "blame";
+    "cat" = "cat";
+    "cleanup" = "cleanup";
+    "clean" = "cleanup";
+    "commit" = "commit";
+    "conflicteditor" = "conflicteditor";
+    "createpatch" = "createpatch";
+    "patch" = "createpatch";
+    "diff" = "diff";
+    "export" = "export";
+    "help" = "help";
+    "ignore" = "ignore";
+    "log" = "log";
+    "merge" = "merge";
+    "pull" = "pull";
+    "push" = "push";
+    "rebase" = "rebase";
+    "refbrowse" = "refbrowse";
+    "reflog" = "reflog";
+    "remove" = "remove";
+    "rm" = "remove";
+    "rename" = "rename";
+    "mv" = "rename";
+    "repocreate" = "repocreate";
+    "init" = "repocreate";
+    "repostatus" = "repostatus";
+    "status" = "repostatus";
+    "resolve" = "resolve";
+    "revert" = "revert";
+    "settings" = "settings";
+    "config" = "settings";
+    "stash" = "stash";
+    "stashapply" = "stashapply";
+    "stashsave" = "stashsave";
+    "subadd" = "subadd";
+    "subsync" = "subsync";
+    "subupdate" = "subupdate";
+    "switch" = "switch";
+    "checkout" = "switch";
+    "sync" = "sync";
+  }
 }
 
 function tgit {
    if($args) {
+    # Replace any aliases with actual TortoiseGit commands
+    if ($Global:TortoiseGitSettings.TortoiseGitCommands.ContainsKey($args[0])) {
+      $args[0] = $Global:TortoiseGitSettings.TortoiseGitCommands.Get_Item($args[0])
+    }
+
     if($args[0] -eq "help") {
       # Replace the built-in help behaviour with just a list of commands
-      $tortoiseGitCommands
-      return    
+      $Global:TortoiseGitSettings.TortoiseGitCommands.Values.GetEnumerator() | sort | Get-Unique
+      return
     }
 
     $newArgs = @()
     $newArgs += "/command:" + $args[0]
-    
+
     $cmd = $args[0]
-    
+
     if($args.length -gt 1) {
       $args[1..$args.length] | % { $newArgs += $_ }
     }
-      
+
     & $Global:TortoiseGitSettings.TortoiseGitPath $newArgs
   }
-}
-
-$tortoiseGitCommands = @(
-"about",
-"log",
-"commit",
-"add",
-"revert",
-"cleanup" ,
-"resolve",
-"switch",
-"export",
-"merge",
-"settings",
-"remove",
-"rename",
-"diff",
-"conflicteditor",
-"help",
-"ignore",
-"blame",
-"cat",
-"createpatch",
-"pull",
-"push",
-"rebase",
-"stashsave",
-"stashapply",
-"subadd",
-"subupdate",
-"subsync",
-"reflog",
-"refbrowse",
-"sync",
-"repostatus"
-) | sort
+}
```

Comment:
EOL @ EOF, please!

(Tip: we have a [.editorconfig](http://editorconfig.org/) that will enforce this for supported editors.)